mrmustard/training/callbacks.py

```diff
@@ -115,7 +117,7 @@ def __post_init__(self):
 
     def get_opt_step(self, optimizer, **kwargs):  # pylint: disable=unused-argument
```

Comment:
As i see it, this function is doing 2 jobs: 
- setting self.optimizer_step
- returning optimizer_step

why not simply replace it by a computed property? and getting rid of the function.
Eg:
class Person():
    def __init__(self):
        self.n = "John Snow"
        self.age = 50
        self.is_old = False if self.age < 25 else True #computed property

Reply:
yes it looks like it could be replaced by a property.